network/src/gossip.rs

```diff
@@ -116,7 +116,7 @@ impl RegisteredMessageValidator {
 	}
 }
 
-// data needed for validating gossip.
+/// The data needed for validating gossip.
 pub(crate) struct MessageValidationData {
```

Comment:
I suppose `MessageValidationData` should be extended in order to convert `ValidatorIndex` from `SignedStatement` to `SessionKey`, right?..

Reply:
yeah, it would have to be